lexicons/atproto.com/repoCreateRecord.json

```diff
@@ -3,15 +3,31 @@
   "id": "com.atproto.repoCreateRecord",
   "type": "procedure",
   "description": "Create a new record.",
-  "parameters": {
-    "did": {"type": "string", "required": true, "description": "The DID of the repo."},
-    "collection": {"type": "string", "required": true, "description": "The NSID of the record collection."},
-    "validate": {"type": "boolean", "default": true, "description": "Validate the record?"}
-  },
   "input": {
     "encoding": "application/json",
-    "description": "The record to create",
-    "schema": {}
+    "schema": {
+      "type": "object",
+      "required": ["did", "collection", "record"],
+      "properties": {
+        "did": {
+          "type": "string",
+          "description": "The DID of the repo."
+        },
+        "collection": {
+          "type": "string",
+          "description": "The NSID of the record collection."
+        },
+        "validate": {
+          "type": "boolean",
+          "default": true,
+          "description": "Validate the record?"
+        },
```

Comment:
Just a note to self (and @dholms I suppose!) to look into xrpc server, ensure it's applying these defaults during validation.  I am pretty sure I witnessed validation not occurring by default when creating a post via curl.

Reply:
Actually you're totally right -- the json schema defaults are _not_ getting applied. I had to modify the repoCreateRecord endpoint to do that manually

Reply:
👍 agreed
general note: let's make issues for stuff like this when we come across it